packages/runtime/src/state/react-page.ts

```diff
@@ -197,27 +165,6 @@ export function getElementPropControllerDescriptors(
   return getComponentPropControllerDescriptors(state, element.type)
 }
 
-export function getElementId(state: State, documentKey: string, elementKey: string): string | null {
```

Comment:
Replaced by a simple [read from the state](https://github.com/makeswift/makeswift/pull/894/files#diff-366e1d38abb10b78254fe441d335387f0bf21463b598e0cbfa26fbd4df140cd8R152).